airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -144,3 +145,17 @@ def set_autocommit(self, conn, autocommit: Any) -> None:
 
     def get_autocommit(self, conn):
         return getattr(conn, 'autocommit_mode', False)
+
+    def run(self, sql, autocommit=False, parameters=None):
+        """
+        Snowflake-connector doesn't allow natively the execution of multiple SQL statements in the same
+        call. So for allowing to pass files or strings with several queries this method is coded,
```

Comment:
@JavierLopezT does this actually support running a file?   It looks like it must be string here.

I thing we absolutely should make it so `sql` can be path _or_ sql, though it doesn't have to be this PR -- I'd just suggest making sure the docstring is consistent here.

 Sorry missed this before.

Reply:
and small nit, `that relies on run from DBApiHook` no longer true